backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/CLIParser.java

```diff
@@ -57,8 +57,7 @@
 
     private static final List<AttachmentType> ACCEPTABLE_ATTACHMENT_TYPES = ImmutableList.of(
```

Comment:
One could ask the parsers for their list of possible attachments. But, since the attachments are defined by the list on https://github.com/sw360/sw360portal/wiki/Dev-Attachment-File-Types. There speaks nothing against the placement of this list at an central place.